pkg/project/project.go

```diff
@@ -49,7 +49,7 @@ func Delete(client *occlient.Client, projectName string) error {
 	return nil
 }
 
-func List(client *occlient.Client) (ProjectList, error) {
+func DescribeProjects(client *occlient.Client) (ProjectList, error) {
 	currentProject := client.GetCurrentProjectName()
 	allProjects, err := client.GetProjectNames()
```

Comment:
You could just reuse List here

Reply:
Is it the same?

The real question is do we need both `GetProjectNames` and `List`. `List` just wraps the error nothing else

Reply:
one and the same

Reply:
* `GetProjectNames` is in occlient.go though

Reply:
updated the code, please review again

Reply:
> * `GetProjectNames` is in occlient.go though

ah, didn't noticed that they are in a different package 😇 , in that case, it might be ok to have both 😇 

But this is also ok ;-)